src/main/scala/com/fortysevendeg/mvessel/metadata/DatabaseMetaData.scala

```diff
@@ -58,23 +58,23 @@ class SQLDroidDatabaseMetaData(connection: Connection)
     table: String,
     scope: Int,
     nullable: Boolean
-    ): ResultSet = bestRowIdStatement.executeQuery()
+    ): SQLResultSet = bestRowIdStatement.executeQuery()
 
-  override def getCatalogs: ResultSet = catalogsStatement.executeQuery()
+  override def getCatalogs: SQLResultSet = catalogsStatement.executeQuery()
 
   override def getColumnPrivileges(
     catalog: String,
     schema: String,
     table: String,
     columnNamePattern: String
-    ): ResultSet = columnPrivilegesStatement.executeQuery()
+    ): SQLResultSet = columnPrivilegesStatement.executeQuery()
 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method is longer than 50 lines](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213819/issues/source?bid=2442814&fileBranchId=2752966#l72)